### PR TITLE
Added the partitionState map in consumer. Waiting the consumer state, if...

### PR DESCRIPTION
**Pull request by chaithu14**
... anyone of the kafka partition of this consumer is active.

### CODE REVIEW

File: contrib/src/main/java/com/datatorrent/contrib/kafka/SimpleKafkaConsumer.java [hunk 1]
```diff
@@ -95,6 +95,9 @@
     private SimpleKafkaConsumer consumer;
     // partitions consumed in this thread
     private final Set<KafkaPartition> kpS;
+    // Partitions consumed in this thread, don't remove the partitions
```
Comment (tweise) [inline contrib/src/main/java/com/datatorrent/contrib/kafka/SimpleKafkaConsumer.java]: Comment is confusing. Please clearly state intent.

File: contrib/src/main/java/com/datatorrent/contrib/kafka/SimpleKafkaConsumer.java [hunk 2]
```diff
@@ -128,6 +132,8 @@ public void run()
           }
         }
 
+        boolean flag = false; // Enable the flag, if anyone of the kafka partitions of this consumer is active
```
Comment (tweise) [inline contrib/src/main/java/com/datatorrent/contrib/kafka/SimpleKafkaConsumer.java]: Name variable to indicate purpose.

**Comment by chaithu14** [comment]
Updated the code as per the review comments.

**Comment by tweise** [comment]
No longer relevant.

**Pull request closed without merging**